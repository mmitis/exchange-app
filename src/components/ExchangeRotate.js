//@flow
import React from "react";

type Props = {
    onClick: () => void
}
export const ExchangeRotate = ({ onClick } : Props) => {
    return (
            <div className="exchange-rotate">
                <button className="exchange-rotate__button" onClick={onClick} >
                    <i className="material-icons">swap_vert</i>
                </button>
            </div>
    )
};

export default React.memo(ExchangeRotate);
