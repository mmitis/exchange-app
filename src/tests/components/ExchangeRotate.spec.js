import ExchangeRotate from './../../components/ExchangeRotate';
import { mount } from 'enzyme';
import React from "react";

describe('Component: ExchangeRotate', () => {

    it('test simple render', async () => {

        const rotate = mount(<ExchangeRotate
            onClick={() => {}}
        />);
        
        expect(rotate).toMatchSnapshot();
    });

});
