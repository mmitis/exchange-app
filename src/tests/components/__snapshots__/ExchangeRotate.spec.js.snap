// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ExchangeRotate test simple render 1`] = `ReactWrapper {}`;
